app/assets/javascripts/area_served.js

```diff
@@ -0,0 +1,45 @@
+
+
+
+function calculate_client_share_total(){
+    total = 0;
+    console.log("*********mark 1 total is: " +total);
+    const client_shares = document.querySelectorAll('*[id$="client_share"]')
```

Comment:
Kind of annoying, but JavaScript variables and functions should be in camelCase instead of snake\_case.

Reply:
Hm.  Fine. Will be corrected in next push.